src/Faker/Provider/ar_SA/Company.php

```diff
@@ -60,4 +62,16 @@ public function bs()
 
         return join($result, ' ');
     }
+
+    /**
+     * example 7001010101
+     **/
+    public static function companyIdNumber()
+    {
+        do {
+            $number = static::numerify('700#######');
```

Comment:
I don't want to sound stupid, but I see the changes on the last commit on this PR.